src/components/shared/StackSettings.js

```diff
@@ -211,7 +215,9 @@ class StackSettingsImpl extends PureComponent<Props> {
                   onChange={this._onInvertCallstackClick}
                   checked={invertCallstack}
                 />
-                {' Invert call stack'}
```

Comment:
~~I see this `{' '}` at the start again, is this necessary ?~~
I checked on production and I don't see a difference with production, so it's good for me. Not sure why this works :-) It's possible that `inline-flex` doesn't use the spaces around the text?

Reply:
Yeah, I also checked and didn't see any difference. Probably.